src/fragments/lib-v1/analytics/ios/identifyuser.mdx

```diff
@@ -1,4 +1,4 @@
-This call sends information that you have specified about the user to Amazon Pinpoint. This could be an authenticated user. If the user is signed in through [Amplify.Auth.signIn](/[platform]/prev/build-a-backend/auth/enable-sign-up/), then you can retrieve the current user and use it with Analytics. You can also provide location information in `AnalyticsUserProfile.Location`.
+This call sends information that you have specified about the user to Amazon Pinpoint. This could be an authenticated user. If the user is signed in through [Amplify.Auth.signIn](/[platform]/prev/build-a-backend/auth/enable-sign-in/), then you can retrieve the current user and use it with Analytics. You can also provide location information in `AnalyticsUserProfile.Location`.
```

Comment:
This one requires a version switcher as depends on if mobile or web. Opened a task to fix all references to enable-sign-up/in